client/src/pages/NeedHelp.js

```diff
@@ -108,6 +131,15 @@ const Step3 = (props) => {
     </WizardStep>
   );
 };
+function validate(values) {
+  let errors = {};
+  if (!values.email) {
+    errors.email = 'Email address is required';
+  } else if (!/\S+@\S+\.\S+/.test(values.email)) {
```

Comment:
There's an existing email validator utility function in `client/src/utils/validators.js` that you can use here. I have a PR [here](https://github.com/FightPandemics/FightPandemics/pull/560/files#diff-c22aa1881b72d5dada9129049a0a36ed) that's going to make some changes to it.

Reply:
Thats great, but we still need to customize the error message for these emails.

Reply:
The utility function returns a Boolean, so instead of implementing your own regex here, call the utility function to validate the email and raise the error string you already have. Implementing your own regex here to validate emails duplicates existing functionality.

Reply:
Good points @BMariscal 
Seems like the regex comes from Auth0 which is good because we want this consistent with backend, API validation and frontend everywhere https://github.com/auth0/lock/blob/4aaefa4a9ed00423e665e0743a7b0a952c6a267d/src/field/email.js#L7 

The react-hook-form library also has support for regular expressions to validate fields but maybe it's better to use the function everywhere since I see now it also adds lowercasing which I'm not sure would be handled by that library.

Reply:
Haven't worked with regex much but I ran into an [post](https://www.loggly.com/blog/regexes-the-bad-better-best/) that talks about how regex can have poor performance. Maybe it's not a worry for us?

Reply:
yeah that's also a good point, from my experience regexes can also get very complex and it's best not to try to do too much with them... 
as far as performance, it also depends how it's used, if the regex is only called to validate once, not too bad, but if it fires for every key stroke etc. then you probably have to optimize that too, e.g. with a debounce